milestone-3/frontend/src/components/recipePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+interface Recipe {
+    name: string;
+    image: string;
+    desc: string;
+}
```

Comment:
this Recipe interface is different than your other Recipe interface in types.. if you have two different interfaces it's good to use different names, even if they are in different files just to avoid confusion.